advisor/2016-07-12-preview/swagger/advisor.json

```diff
@@ -0,0 +1,461 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"version": "v1",
+		"title": "Microsoft.Azure.Engineering.Gateway"
+	},
+	"host": "management.azure.com",
+	"schemes": ["https"],
+	"paths": {
+		"/subscriptions/{subscriptionId}/providers/microsoft.Advisor/generaterecommendations": {
+			"post": {
+				"tags": ["GenerateRecommendations"],
+				"operationId": "GenerateRecommendations_POST",
+				"consumes": [],
+				"produces": ["application/json",
```

Comment:
Rule S2004 from https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md

Produces and Consumes global arrays SHOULD have "application/json" as the supported MIME-TYPE. If the operation needs to have a different one then it can override it.